docs/openapi.yaml

```diff
@@ -2064,6 +2064,25 @@ components:
             - STOPPED
         pricing:
           $ref: '#/components/schemas/PricingUpdateRequest'
+        interval:
+          type: object
+          description: |-
+            The interval of the agreement.
+            
+            The interval is specified by the `type` and `period` properties.
+            When the type is `RECURRING`, then the property `period` is required.
+            When the type is `FLEXIBLE`, then the property `period` is not allowed.
```

Comment:
I think we should remove this for now. And when we enable FLEXIBLE intervals, we add it here as well.